libs/ui-lib-tests/cypress/integration/use-cases/create-cluster/with-external-partner-integrations.cy.ts

```diff
@@ -42,6 +43,12 @@ describe('Create a new cluster with external partner integrations', () => {
       ClusterDetailsForm.customManifestsControl.findCheckbox().should('be.checked');
     });
 
+    it('External partner integrations checkbox is unselected after OCP < v4.14 is selected', () => {
```

Comment:
You'll need to add the `EXTERNAL_PLATFORM_OCI` entry to the mocks in `libs/ui-lib-tests/cypress/fixtures/infra-envs/feature-support.ts` for this test to work properly.